.php_cs.cache

```diff
@@ -1 +1 @@
215647660,"src\/Service\/CancelService.php":2005833364,"src\/Service\/UpgradeNoticeService.php":278481121,"src\/Service\/IssuerService.php":2436739866,"src\/Service\/OrderStatusService.php":238839263,"src\/Service\/RepeatOrderLinkFactory.php":3345999482,"src\/Service\/RefundService.php":1899926352,"src\/Service\/CustomerService.php":2318057032,"src\/Service\/ShipmentService.php":4251995706,"src\/Service\/MolCarrierInformationService.php":4293776795,"src\/Service\/MolliePaymentMailService.php":172947755,"src\/Service\/ApiService.php":79491916,"src\/Service\/SettingsSaveService.php":1730766643,"src\/Service\/LanguageService.php":2124207838,"src\/Service\/OrderCartAssociationService.php":2088165512,"src\/Service\/MemorizeCartService.php":1664458900,"src\/Service\/ErrorDisplayService.php":1143931943,"src\/Service\/ShipService.php":1602580831,"src\/Service\/ConfigFieldService.php":3478838713,"src\/Service\/PaymentMethod\/PaymentMethodSortProviderInterface.php":3377687683,"src\/Service\/PaymentMethod\/PaymentMethodSortProvider.php":2516837700,"src\/Service\/TransactionService.php":3012116315,"src\/Service\/OrderFeeService.php":1157946833,"src\/Service\/index.php":1565892783,"src\/Service\/CartLinesService.php":4076675848,"src\/Service\/PaymentMethodService.php":779107452,"src\/Service\/CartDuplicationService.php":2083001411,"src\/Service\/RestorePendingCartService.php":1975879317,"src\/Service\/CarrierService.php":3025184296,"src\/Service\/MollieOrderInfoService.php":4246642683,"src\/Service\/ImageService.php":2427608122,"src\/Service\/ApiKeyService.php":1611601228,"src\/Service\/CountryService.php":10428413,"src\/Service\/ExceptionService.php":1863518302,"src\/Config\/Config.php":1752507235,"src\/Config\/index.php":1565892783,"src\/Validator\/NewOrderMailValidator.php":2724552544,"src\/Validator\/index.php":4125643355,"src\/Validator\/VoucherValidator.php":2478334963,"src\/Validator\/OrderConfMailValidator.php":4035002233,"src\/Validator\/MailValidatorInterface.php":870550877,"src\/index.php":1565892783,"src\/Entity\/MolCustomer.php":3967015682,"src\/Entity\/MolCarrierInformation.php":828974673,"src\/Entity\/MolPaymentMethodIssuer.php":394582856,"src\/Entity\/MolPaymentMethod.php":3669138095,"src\/Entity\/index.php":1565892783,"src\/Entity\/MolPendingOrderCart.php":3634621286,"src\/Entity\/MolOrderFee.php":3242051293,"src\/DTO\/PaymentData.php":399204546,"src\/DTO\/index.php":4125643355,"src\/DTO\/OrderData.php":1529412272,"src\/DTO\/Object\/Amount.php":2059362527,"src\/DTO\/Object\/index.php":4125643355,"src\/DTO\/Line.php":1278234341,"src\/Controller\/index.php":4125643355,"src\/Controller\/AbstractMollieController.php":2421616576,"src\/Provider\/AbstractCustomLogoProvider.php":381061964,"src\/Provider\/PhoneNumberProvider.php":3332886073,"src\/Provider\/PhoneNumberProviderInterface.php":1318893057,"src\/Provider\/CreditCardLogoProvider.php":490197597,"src\/Provider\/index.php":1565892783,"src\/Provider\/CustomLogoProviderInterface.php":1715125740}}
```

Comment:
this can be gitignored

Reply:
Add it into .gitignore or just not commit it?